test/advanced.test.js

```diff
@@ -277,6 +287,21 @@ describe("mithril-objectify", function() {
                 '({tag:"div",attrs:undefined,children:[JSON.stringify({})],dom:undefined,domSize:undefined,events:undefined,key:undefined,state:{},text:undefined});'
             );
         });
+
+        it("should not convert JSON when entries are not objects", function() {
```

Comment:
Why is `JSON.stringify("string")` not safe to optimize? It's still a string, and thus mithril will treat it as a child text node.